civis/response.py

```diff
@@ -131,8 +134,11 @@ def __init__(self, json_data, snake_case=True, headers=None):
                 else:
                     val = v
 
-                self.update({key: val})
-                self.__dict__.update({key: val})
+                self_updates[key] = val
+
+        self.update(self_updates)
+        # Update self.__dict__ at the end to avoid replacing the update method.
+        self.__dict__.update(self_updates)
```

Comment:
so the issue is that the `update` key was overwriting the `dictionary` `update` method?

Reply:
yeah, and then the next `update` call would fail.